src/server/server.cc

```diff
@@ -1407,7 +1407,8 @@ time_t Server::GetLastScanTime(const std::string &ns) {
   return 0;
 }
 
-void Server::SlowlogPushEntryIfNeeded(const std::vector<std::string> *args, uint64_t duration) {
+void Server::SlowlogPushEntryIfNeeded(const std::vector<std::string> *args, uint64_t duration, std::string client_name,
```

Comment:
Is it better to pass the `const Connection *conn` here? instead of passing its member one by one.